scripts/completion/dvc.bash

```diff
@@ -24,26 +24,26 @@ _dvc_destroy='-f --force'
 _dvc_diff='-t --show-json --show-hash'
 _dvc_fetch='-j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -R --recursive'
 _dvc_fetch_COMPGEN=_dvc_compgen_DVCFiles
-_dvc_get_url=''
+_dvc_gc='-a --all-branches -T --all-tags -w --workspace -c --cloud -r --remote -f --force -p --projects -j --jobs'
 _dvc_get='-o --out --rev --show-url'
-_dvc_gc='-a --all-branches --all-commits -T --all-tags -w --workspace -c --cloud -r --remote -f --force -p --projects -j --jobs'
-_dvc_import_url='-f --file'
+_dvc_get_url=''
 _dvc_import='-o --out --rev'
+_dvc_import_url='-f --file'
 _dvc_init='--no-scm -f --force'
 _dvc_install=''
 _dvc_list='-R --recursive --outs-only --rev'
 _dvc_list_COMPGEN=_dvc_compgen_files
-_dvc_lock='$(compgen -f -X \!*?.dvc)'
-_dvc_metrics='add modify rmeove show'
+_dvc_lock=_dvc_compgen_DVCFiles
+_dvc_metrics='add diff modify remove show'
 _dvc_metrics_add='-t --type -x --xpath'
 _dvc_metrics_add_COMPGEN=_dvc_compgen_files
-_dvc_metrics_show='$(-t --type -x --xpath -a --all-branches -T --all-tags -R --recursive'
-_dvc_metrics_show_COMPGEN=_dvc_compgen_files
 _dvc_metrics_diff='--targets -t --type -x --xpath -R --show-json'
 _dvc_metrics_modify='-t --type -x --xpath'
 _dvc_metrics_modify_COMPGEN=_dvc_compgen_files
-_dvc_metrics_remove='$(compgen -f)'
-_dvc_move='$(compgen -f)'
+_dvc_metrics_remove=_dvc_compgen_files
+_dvc_metrics_show='$(-t --type -x --xpath -a --all-branches -T --all-tags -R --recursive'
```

Comment:
Does this not require matching parens? :confused:

Reply:
not really; requires no parens